MatiasDure/Project_IMTS
# Bee idle fly

**Description**

- Implement bee idle flying
- Implement placeholder village
- Bee idle fly when scene village tracked image

**ScreenShot**
![bee idle flying](https://github.com/user-attachments/assets/a2482019-83ab-46ff-803e-90462ae8665e)

**How to test**

- Open scene SetUpVillage
- Play around with the village plot
- Check if anything break
